refbox/src/refBox.js

```diff
@@ -100,11 +82,29 @@ class SettingSelector extends React.Component {
   render() {
     if (this.state.configuring) {
       return this.renderOptions();
+    } else if (this.props.current.length === 0 || !this.props.options.includes(this.props.current)) {
+      return this.renderNoSelection();
     } else {
       return this.renderCurrent();
     }
   }
 
+  renderNoSelection() {
+    // const description = this.props.prefix ? this.props.prefix + this.props.current : this.props.current;
+    // const pending = this.state.current && this.props.current !== this.state.current;
```

Comment:
Why is this here? Is the future or past code?

Reply:
Result of copy-pasting :see_no_evil: , removed in 2efab9f7c913e5ba7c1d57dafde1be8a46f90069